package/MDAnalysis/lib/distances.py

```diff
@@ -85,6 +124,13 @@
 from .c_distances import _minimize_vectors_ortho, _minimize_vectors_triclinic
 
 
+try:
+    import distopia
```

Comment:
As far as I can tell you don't actually use this import directly anywhere right? Can you not just move this try/except down to 146 and just feel the _distances dictionary directly?

Reply:
The aim was to have the `HAS_DISTOPIA` attr available for backend selection so I wouldn't mind keeping it in? On the other hand we are removing the `backend: str = "distopia" if HAS_DISTOPIA else "serial"` but. IDK if people feel strongly about YAGNI  I can pull it out.

Reply:
Its also needed for the tests as module level import to avoid repetition.

Reply:
I don't understand why you need two try/excepts (which aren't exactly cheap if we assume the base case is that you'll hit the error most of the time).

Why can't we move the `HAS_DISTOPIA` assignment under with the same imporlib section on line 156? i.e.

```python
try:
    _distances["distopia"] = importlib.import_module("._distopia", package="distopia")
except:
    HAS_DISTOPIA = False
else:
    HAS_DISTOPIA = True
```